src/ReCaptcha.php

```diff
@@ -17,40 +16,34 @@
  */
 class ReCaptcha
 {
-    /**
-     * URI to the regular API
-     *
-     * @var string
-     */
-    const API_SERVER = 'http://www.google.com/recaptcha/api';
 
     /**
-     * URI to the secure API
+     * URI to the API
      *
      * @var string
      */
-    const API_SECURE_SERVER = 'https://www.google.com/recaptcha/api';
```

Comment:
@Maks3w Done on both urls, following the documentation (The verify url is also https).